### PR TITLE
Build/eslint fix

### PR DESCRIPTION
**Pull request by ccamel**
Self explanatory - fix `eslint` arguments that prevented from working properly.

### CODE REVIEW

**Review by lolottetheclash — APPROVED** [review]

File: src/ui/view/governance/details/details.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const Paragraph: FC<ParagraphProps> = ({ paragraph, theme }) => {
   const isTextTooLong = useMemo(
     () =>
       paragraphRef.current && paragraphRef.current.offsetHeight < paragraphRef.current.scrollHeight,
-    [paragraphRef.current]
```
Comment (0xLodz) [inline src/ui/view/governance/details/details.tsx:51]: Good catch. We can refactor it using useCallback [as suggested here](https://legacy.reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node) to remove this unnecessary ref dependency in another PR.

**Review by 0xLodz — APPROVED** [review]

File: src/ui/view/governance/details/details.tsx (hunk 1 above)
Comment (ccamel) [reply src/ui/view/governance/details/details.tsx:51]: All right. Can you make the change? thanks :)

Comment (0xLodz) [reply src/ui/view/governance/details/details.tsx:51]: Sure!

Comment (ccamel) [reply src/ui/view/governance/details/details.tsx:51]: Done by d05aa339492b70d12afe8ed41e05dc8e03cb8d73.

**Pull request merged**